diff --git a/src/commands/title3.tsx b/src/commands/title3.tsx
index bde067d01ae1c306a1bd89b064edb3a985fb569d..fa3bfb8df3f8299ef9e790e170887a382ba7a5c4 100644
--- a/src/commands/title3.tsx
+++ b/src/commands/title3.tsx
@@ -11,13 +11,13 @@ export const heading3: ICommand = {
   buttonProps: { 'aria-label': 'Insert Heading 3 (ctrl + 3)', title: 'Insert Heading 3 (ctrl + 3)' },
   icon: <div style={{ fontSize: 15, textAlign: 'left' }}>Heading 3</div>,
   execute: (state: ExecuteState, api: TextAreaTextApi) => {
-    headingExecute({ state, api, prefix: state.command.prefix!, suffix: state.command.suffix });
+    headingExecute({ state, api, prefix: state.command.prefix ?? '### ', suffix: state.command.suffix ?? '' });
   },
 };
 
 /**
- * @deprecated Use `heading3` instead.
- * This command is now deprecated and will be removed in future versions.
- * Use `title3` for inserting Heading 3.
+ * @deprecated Since v4.0.0. Use `heading3` instead.
+ * Scheduled for removal in v5.0.0.
+ * @see heading3
  */
 export const title3: ICommand = heading3;
